### PR TITLE
Components

### PR DESCRIPTION
**Pull request by abenezer-addisu**
On this PR I have done the following:

- [ ] Created a new directory called Components.
- [ ] Created a new file called Calculator.js a React component that matches the given design
- [ ] Used functional components

### CODE REVIEW

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @abenezer-addisu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Gitfloiw
✅ No linter error


## Required Changes ♻️
- [ ] Kindly fix the link to your `License` as it is currently broken, This will make your readme file more professional

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      <div className="buttons">
+        <button type="button">AC</button>
+        <button type="button">+/-</button>
+        <button type="button">% </button>
+        <button type="button" className="operator">
+          ÷
+        </button>
+        <button type="button">7</button>
+        <button type="button">8</button>
+        <button type="button">9</button>
```
Comment (ghost) [inline src/components/Calculator.js]: - [ ] Good work up to this point but there is a project requirement you have not met yet. The calculator component should have at least 1 child component. kindly implement that

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @abenezer-addisu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

✅ Correct use of Gitflow
✅ No linter error


## Required Changes ♻️

- [ ] Kindly fix the link to your ` License` as it is currently broken. This will make your readme file more professional

_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by victorokoroji — APPROVED** [review]
Hi @abenezer-addisu, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- Requested change implemented ✅ 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**